app/main.py

```diff
@@ -1,3 +1,56 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
```

Comment:
`km` could be `float` as well, add it in the type annotation here and across other methods where `int` is used.